webhook/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Don't commit log files
+logs/
+
+# This file is encrypted
```

Comment:
Please update comment.

Reply:
Updated in dbab857d611c1d6aaf18033fba41217c69a193e2